airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSourceOperations.java

```diff
@@ -140,6 +144,18 @@ protected void putGeography(final ObjectNode node,
     node.put(columnName, Geography.deserialize(resultSet.getBytes(index)).toString());
   }
 
+  @Override
+  protected void putTimestamp(final ObjectNode node, final String columnName, final ResultSet resultSet, final int index) throws SQLException {
+    final DateTimeFormatter microsecondsFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss[.][SSSSSS]");
+    try {
+      node.put(columnName, getObject(resultSet, index, LocalDateTime.class).format(microsecondsFormatter));
+    } catch (final Exception e) {
+      // for backward compatibility
```

Comment:
what does that mean? Should we at least log something here ?

Reply:
After thinking about it, I'm going to remove this section altogether. This was done inside of `AbstractJdbcCompatibleSourceOperations#putTimestamp` for generic JDBC sources because we didn't know if we were able to parse the result to the appropriate `Datetime/Date/Timestamp` Java type. But since we know how a MSSQL datetime result looks like, this isn't necessary.